shell/platform/darwin/macos/framework/Source/FlutterExternalTextureMetal.mm

```diff
@@ -54,11 +54,13 @@ - (BOOL)populateTexture:(FlutterMetalExternalTexture*)textureOut {
 
   if (cvReturn != kCVReturnSuccess) {
     NSLog(@"Could not create Metal texture from pixel buffer: CVReturn %d", cvReturn);
+    CVPixelBufferRelease(pixelBuffer);
```

Comment:
Can we wrap these in `fml::CFRef` please?

Reply:
Good idea! Done.